templates/books-show.html

```diff
@@ -0,0 +1,20 @@
+<div ng-model="book in booksShowCtrl" class="container">
```

Comment:
The "in" syntax is usually used for ng-repeat, not for ng-model, and I don't think you need any ng-model on this element.